templates/CRM/common/TabHeader.js

```diff
@@ -148,4 +155,25 @@
       $panel.crmSnippet('destroy');
     }
   };
+
+  /**
+   * Updates the query parameter in the page URL,
+   * or adds the parameter if its not currently there.
+   *
+   * @param {string} param
+   * @param {string} value
+   * @return void
+   */
+  function updateUrlParameter(param, value) {
+    var href = window.location.href;
```

Comment:
It's bit of a no-no in javascript to declare a `var` more than once. I would just declare it at the top and then assign values to it below.
```suggestion
    var newUrl,
      href = window.location.href;
```